airbyte-integrations/connectors/source-mssql/src/main/resources/spec.json

```diff
@@ -111,6 +111,20 @@
         "default": "STANDARD",
         "enum": ["STANDARD", "CDC"],
         "order": 8
+      },
+      "is_snapshotDisabled": {
```

Comment:
Let's not mix camel case and snake case in the same identifier. Property names have so far been following snake case, so let's stick with that

Reply:
followed only snake case.